CHANGELOG.md

```diff
@@ -2,6 +2,28 @@
 
 <!-- prettier-ignore-start -->
 
+## 0.8.15
+
+### Enhancements
+
+- Add `uv auth` commands ([#15570](https://github.com/astral-sh/uv/pull/15570))
```

Comment:
Maybe I'd enumerate these? Or append something more? Like "to manage credentials"?